src/alexapi/triggers/platformtrigger.py

```diff
@@ -0,0 +1,41 @@
+from basetrigger import BaseTrigger
+import alexapi.triggers as triggers
+
+
+class PlatformTrigger(BaseTrigger):
+
+	type = triggers.TYPES.OTHER
+
+	_platform_continuous_callback = None
+
+	def __init__(self, config, trigger_callback):
+
+		super(PlatformTrigger, self).__init__(config, trigger_callback, 'platform')
+
+		self._event_type = triggers.EVENT_TYPES.ONESHOT if self._tconfig['event_type'] == 'oneshot' \
```

Comment:
I cannot see `_event_type` anywhere else. Should this be `event_type` to match the parent class?